### PR TITLE
test: config route with management fileds (E2E)

### LINKED ISSUE #638: add e2e test for config route with management fileds
detail:

 - [ ]  add test case for config route with name and desc

 - [ ]  verify the route just configured with correct name and desc, and has create_time and update_time

 - [ ]  add test case for update route  with other name and desc

 - [ ]  verify the route just configured with correct name and desc, update_time has changed.

 - [ ]  add test case for config route with labels and filter_func

 - [ ]  add test case for config route with  non-existent fields

 - [ ]  verify routing correctness under multi routes

For background and description, please see:

https://github.com/apache/apisix-dashboard/issues/607

### PR DESCRIPTION
**Pull request by idbeta**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] E2E test
- [ ] New feature provided
- [ ] Improve performance

- Related issues
resolve #638

### CODE REVIEW

File: api/test/e2e/route_with_management_fileds_test.go [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+			Headers:      map[string]string{"Authorization": token},
+			ExpectStatus: http.StatusOK,
+		},
+	}
+
+	for _, tc := range tests {
+		testCaseCheck(tc)
+	}
+
+	tests = []HttpTestCase{
```
Comment (membphis) [inline api/test/e2e/route_with_management_fileds_test.go:162]: I think we should put them to a new function

**Review by membphis — CHANGES_REQUESTED** [review]

File: api/test/e2e/route_with_management_fileds_test.go (hunk 1 above)
Comment (idbeta) [reply api/test/e2e/route_with_management_fileds_test.go]: fixed.

**Review by membphis — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
@nic-chen @gxthrj @moonming please take a look when you have time

**Comment by nic-chen** [comment]
LGTM Except for some code format issues.

**Review by nic-chen — APPROVED** [review]

**Pull request merged**